python/mpt/tests/test_utils.py

```diff
@@ -1,34 +1,34 @@
-from pathlib import Path
+from ethereum.cancun.trie import BranchNode, ExtensionNode, InternalNode
 
-import pytest
-from ethereum.cancun.trie import BranchNode, ExtensionNode
-from ethereum_types.bytes import Bytes
+from mpt.trie_diff import resolve
 
-from mpt.utils import decode_node
 
+def test_embedded_nodes_resolve(branch_in_extension_data):
```

Comment:
useless test, remove it and test the actual diff

Reply:
you should use your `branch_in_extension_data` as a parameter inside `test_resolve`. And